watertap/costing/watertap_costing_package.py

```diff
@@ -1317,3 +1357,37 @@ def cost_by_flow_volume(blk, flow_cost, flow_to_cost):
     blk.capital_cost_constraint = pyo.Constraint(
         expr=blk.capital_cost == blk.flow_cost * flow_to_cost
     )
+
+
+def cost_uv_aop_bundle(blk, reactor_cost, lamp_cost, factor_uv_replacement):
+    """
+    Generic function for costing a UV system.
+
+    Args:
+        reactor_cost - The cost of UV reactor in [currency]/[volume]
+        lamp_cost - The costs of the lamps, sleeves, ballasts and sensors in [currency]/[kW]
+    """
+    make_capital_cost_var(blk)
+    make_fixed_operating_cost_var(blk)
+    blk.reactor_cost = pyo.Expression(expr=reactor_cost)
+    blk.lamp_cost = pyo.Expression(expr=lamp_cost)
+    blk.factor_uv_replacement = pyo.Expression(expr=factor_uv_replacement)
+
+    flow_in = pyo.units.convert(
+        blk.unit_model.control_volume.properties_in[0].flow_vol,
+        to_units=pyo.units.m**3 / pyo.units.hr,
+    )
+
+    electricity_demand = pyo.units.convert(
+        blk.unit_model.electricity_demand[0], to_units=pyo.units.kW
+    )
+
+    blk.capital_cost_constraint = pyo.Constraint(
+        expr=blk.capital_cost
+        == blk.reactor_cost * flow_in * blk.unit_model.num_of_reactors
+        + blk.lamp_cost * electricity_demand
+    )
+    blk.fixed_operating_cost_constraint = pyo.Constraint(
+        expr=blk.fixed_operating_cost
+        == blk.factor_uv_replacement * blk.lamp_cost * electricity_demand

```

Comment:
Can we rename this `factor_uv_lamp_replacement` so that it is immediately obvious?

Reply:
Yeah, I think it is a good idea to be more clear